src/modules/navigation/duck/actions.jsx

```
@@ -4,11 +4,13 @@ import { isDirectory } from 'cozy-client/dist/models/file'
 import flag from 'cozy-flags'
 import { QuotaPaywall } from 'cozy-ui-plus/dist/Paywall'
 
-import { ROOT_DIR_ID, TRASH_DIR_ID } from '@/constants/config'
-import { MAX_PAYLOAD_SIZE_IN_GB } from '@/constants/config'
+import { resetQuery as resetQueryAction } from 'cozy-client/dist/store'
+
+import { ROOT_DIR_ID, TRASH_DIR_ID, MAX_PAYLOAD_SIZE_IN_GB } from '@/constants/config'
```

ESLint

[error] Replace `·ROOT_DIR_ID,·TRASH_DIR_ID,·MAX_PAYLOAD_SIZE_IN_GB·` with `⏎··ROOT_DIR_ID,⏎··TRASH_DIR_ID,⏎··MAX_PAYLOAD_SIZE_IN_GB⏎`

(prettier/prettier)